src/components/avatar/pages/avatar-interactive.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
that's how doc section for Avatars will look when we switch to interactive components